x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -1635,6 +1636,112 @@ var sysmon = (function () {
         .Add(removeEmptyEventData)
         .Build();
 
+    // Event ID 24 - ClipboardChange (New content in the clipboard).
+    var event24 = new processor.Chain()
+        .Add(parseUtcTime)
+        .AddFields({
+            fields: {
+                category: ["file"], // pipes are files
```

Comment:
I am not sure if we can consider the clipboard a file, any ideas?

Reply:
IMO it's not file related unless it's a file copy-paste operation (which are not recorded by sysmon IIUC).

Another argument you could make is that when it is archived that it does create a file, but that seems more like meta info.

Reply:
Then I am not sure what category put it in, I do not see any in https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html that feels evident to me, wdyt?

Reply:
I agree, none of the categories fit. I don't think there's an ECS requirement to populate it so I'd omit it.